static/webhook-ips.json

```diff
@@ -3,12 +3,15 @@
     "44.228.126.217",
     "50.112.21.217",
     "52.24.126.164",
-    "54.148.139.208"
+    "54.148.139.208",
+    "2600:1f24:64:8000::/52",
+    "2600:1f28:37:4000::/52"
```

Comment:
As mentioned elsewhere: please split use1 and US, and mark (e.g. in parenthesis in the title) use1 as "private" or something.